src/bidiMapper/domains/context/bidiPreloadScript.ts

```diff
@@ -74,7 +74,7 @@ export class BidiPreloadScript {
     this.#channels =
       params.arguments?.map((a) => new ChannelProxy(a.value)) ?? [];
     this.#functionDeclaration = params.functionDeclaration;
-    this.#contextId = params.context ?? null;
+    this.#contextId = params.sandbox ?? null;
```

Comment:
I don't think you can use sandbox as a contextId. I think we need to fix the mapper logic first: the spec does not define a context support for preload scripts and sandboxes is a separate feature that we don't yet support in the mapper for preload scripts.